cves/kernel/CVE-2014-0196.yml

```diff
@@ -270,8 +268,8 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: This is because the race condition in this vulnerability can overflow the buffer where it can give the user admin permissions that they shouldn't have
```

Comment:
A bit of an awkward sentence. Maybe explain in a way that doesn't rely on the context of the question itself as much,